DEPLOYMENT.md

```diff
@@ -1,48 +1,131 @@
 
-   ## Google Play Store (Android)
+## TestFlight (iOS)
 
-2. Add `key.json` and `my-release-key.keystore` files into the `/app/android/` directory. These files contain credentials for uploading the app to the Google Play Store. Ask on Slack for another developer in the team to send you these files. Also ask them for the password for the `my-release-key.keystore` file -- save this somewhere safe (e.g. [a password manager](https://www.techradar.com/uk/best/password-manager)), you'll need it in the future to deploy the app.
+3. Ask Joanna to give you Developer access to the STA App Store Connect account (you should be able to do this without having to pay for an Apple Developer account). This is necessary in order to submit iOS versions of the app via Expo Application Services, and will allow you to check whether releases you submit have been uploaded successfully and accepted by Apple.
 
-   > Because these files contain sensitive access credentials we should never commit them to GitHub as our repository is open-source, anyone can see it.
+# App deployment
 
-   > On some systems, the terminal has a problem if the password for `my-release-key.keystore` contains symbols, so this password may need to be letters and numbers only (just make sure it's a long, strong password). If you need to change the password locally you can use this command `keytool -storepasswd -keystore path/to/my-release-key.keystore -storetype PKCS12`
+**All devs on the team can deploy the app** -- it should be something we all usually incorporate into any pull request where we make a change to the front-end app.
 
-3. Ask Joanna to give you Developer access to the STA Google Play Store account. That will allow you to check whether releases you deploy have uploaded successfully, and you'll be able to add new testers.
+**We use [Expo Application Services (EAS)](https://expo.dev/eas)** to build and deploy the app. We are using the free tier which gives us 30 builds per calendar month (15 iOS, 15 Android -- so it's really 15 new deployments). This includes test builds (the Google Play test version and TestFlight) and production builds, it's all the same total.
 
-## TestFlight (iOS)
+**As a general rule you're strongly encouraged to deploy changes as part of any pull request that updates the front-end app** by following the steps below. **But we also just need to be careful not to hit the 15 deployments (30 builds)) limit,** so if there has been a high frequency of approved pull requests this month or your pull request is only a very minor change to the front-end app, you might not want to update the app version and do a new deployment.  If you're not sure, check [how many builds have already been done this month](https://expo.dev/accounts/scottishtecharmy/settings/billing) (and if you're still unsure ask the team on Slack).
```

Comment:
This is my suggestion of how we approach deployments in the team -- I feel it would be healthy to shift from how things currently are (only a few people doing deployments) to it being part and parcel of what we all do.  It may mean a little juggling within the 15 deployments a month limit, but I feel that's better than the alternatives (only a select few people deploy or e.g. we do one deployment a week which needs its own PR, which feels faffy to me).

But that's just my suggestion.  Very happy to discuss.